### PR TITLE
Initial version of docs generator

### LINKED ISSUE #8: Improve docs
User should be able to read docs and understand what resources are implemented, what fields are exposed (or NOT exposed) and special behavior (when an application resource is added does it start the pipeline?)

We might be able to autogenerate some of this? https://github.com/segmentio/terraform-docs

### PR DESCRIPTION
**Pull request by umairidris**
Fix #8

### CODE REVIEW

File: docs/r/cdap_profile.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# cdap_profile
```
Comment (xingao267) [inline docs/r/cdap_profile.md]: Added a note at the top that these documents are auto-generated and should not be modified manually?

File: tools/generate_docs/generate.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
+package main
```
Comment (xingao267) [inline tools/generate_docs/generate.go]: License

File: docs/r/cdap_namespace.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# cdap_namespace
+
+## Argument Reference
+
+The following fields are supported:
+
+* name:
```
Comment (xingao267) [inline docs/r/cdap_namespace.md]: what about 

name (Required):
The name of the namespace.

Comment (umairidris) [reply docs/r/cdap_namespace.md]: Done.

File: tools/generate_docs/generate.go (hunk 2 above)
Comment (umairidris) [reply tools/generate_docs/generate.go]: Done.

**Pull request merged**